cdap-app-fabric/src/test/java/co/cask/cdap/app/runtime/monitor/RuntimeMonitorTest.java

```diff
@@ -91,6 +92,7 @@
   private DatasetService datasetService;
   private DatasetFramework datasetFramework;
   private Transactional transactional;
+  private RemoteExecutionLogProcessor noOpLogProcessor;
```

Comment:
instead of having a field, it's actually easier to read to do it in-place.

`new RuntimeMonitor(...., m -> { });`